### ISSUE TITLE
TabError in pfam.py, multiple lines (Python-3.4)

### ISSUE THREAD

**Issue by sagitter** [prody/ProDy#249]
Hi all,

this error comes out during the byte compilation with Python-3.4:

```
Bytecompiling .py files below /home/sagitter/rpmbuild/BUILDROOT/ProDy-1.6-3.fc22.x86_64/usr/lib64/python3.4 using /usr/bin/python3.4
*** Error compiling '/home/sagitter/rpmbuild/BUILDROOT/ProDy-1.6-3.fc22.x86_64/usr/lib64/python3.4/site-packages/prody/database/pfam.py'...
Sorry: TabError: inconsistent use of tabs and spaces in indentation (pfam.py, line 72)
```

**Comment by sagitter** [comment]
Temporary patch to remove all TABs: http://fpaste.org/271955/

**Comment by sagitter** [comment]
Patch that I have used (1 tab becomes 4 spaces):

```
--- prody/database/pfam.orig.py	2015-05-29 17:28:39.000000000 +0200
+++ prody/database/pfam.py	2015-09-27 12:57:50.775350991 +0200
@@ -4,7 +4,7 @@
 __author__ = 'Anindita Dutta, Ahmet Bakan, Cihan Kaya'
 
 import re
-import urllib
+import urllib, urllib3
 from os.path import join, isfile
 
 from prody import LOGGER, PY3K
@@ -71,9 +71,9 @@
         fseq = '>Seq\n' + seq
         parameters = { 'hmmdb' : 'pfam', 'seq': fseq }
         enc_params = urllib.urlencode(parameters)
-        request = urllib.request.Request('http://hmmer.janelia.org/search/hmmscan', enc_params)
+        request = urllib.request3.Request('http://hmmer.janelia.org/search/hmmscan', enc_params)
 
-        url = ( urllib.request.urlopen(request).geturl() + '?output=xml') 
+        url = ( urllib3.request.urlopen(request).geturl() + '?output=xml') 
         LOGGER.debug('Submitted Pfam search for sequence "{0}...".'
                      .format(seq[:MINSEQLEN]))
 
--- prody/utilities/pathtools.orig.py	2015-05-29 17:28:39.000000000 +0200
+++ prody/utilities/pathtools.py	2015-09-27 15:01:31.737332379 +0200
@@ -390,7 +390,7 @@
     for opening the *url*."""
 
     try:
-        from urllib2 import urlopen, URLError, Request
+        from urllib3 import urlopen, URLError, Request
     except ImportError:
         from urllib.request import urlopen, Request
         from urllib.error import URLError
--- prody/atomic/subset.orig.py	2015-11-09 20:30:52.000000000 +0100
+++ prody/atomic/subset.py	2015-12-10 18:47:59.172394862 +0100
@@ -33,8 +33,8 @@
         if not isinstance(indices, np.ndarray):
             indices = np.array(indices, int)
         elif not indices.dtype == int:
-	    indices = list(indices[0])
-	    indices = np.array(indices, int)
+            indices = list(indices[0])
+            indices = np.array(indices, int)
 	
 
         if kwargs.get('unique'):
--- prody/sequence/analysis.orig.py	2015-11-09 20:30:52.000000000 +0100
+++ prody/sequence/analysis.py	2015-12-10 18:52:15.485776389 +0100
@@ -576,5 +576,5 @@
 
     toc1 = timeit.default_timer()
     elapsed1 = toc1 - tic1
-    print elapsed1
+    print("elapsed1")
```

**Comment by cihankayacihan** [comment]
This issue is fixed in v1.8.
